mls-rs-crypto-webcrypto/js/node-crypto.js

```diff
@@ -0,0 +1,5 @@
+const crypto = require("node:crypto");
```

Comment:
Does node:crypto have the exact same api as subtle crypto?

Reply:
It seems to me, but I was re-checking the documentation and saw that the correct way is to use:
```suggestion
const crypto = require("node:crypto").webcrypto;
```
I'll test this and change accordingly